modules/github/standardized-repository-settings/main.tf

```diff
@@ -0,0 +1,145 @@
+/**
+  * This module sets up a GitHub repository in a standardized way, to ensure
+  * consistency among things like branch and PR handling, labels and so on.
+  */
+
+terraform {
+  required_version = "0.15.3"
+
+  required_providers {
+    github = {
+      source  = "integrations/github"
+      version = "4.18.0"
+    }
+  }
+}
+
+provider "github" {
+  owner = "XenitAB"
+  token = var.github_token
+}
+
+resource "github_repository" "this" {
+  name        = var.repository_name
+  description = var.repository_description
+
+  allow_auto_merge       = false
+  allow_merge_commit     = true
+  allow_rebase_merge     = true
+  allow_squash_merge     = true
+  delete_branch_on_merge = true
+  has_downloads          = false
+  has_issues             = true
+  has_wiki               = false
```

Comment:
For `has_downloads` that's actually deprecated so that should not be true. My thought was that we always want to have issue reporting (when would we turn it off?) and I'm not sure if GitHub's wiki functionality is something we ever want, so that was my rationale for not turning it on.

Reply:
I don't see a need for variables in this case. We are opinionated in all our terraform modules and if we need it as a variable I think we should add it then.